src/comp_loinc/module.py

```diff
@@ -20,41 +25,44 @@
 
 
 class Module:
+    """Module"""
     from comp_loinc import Runtime
 
     def __init__(self, *, name: str, runtime: Runtime):
         from comp_loinc import Runtime
-
+        logger.info(f"Starting: {name}")
```

Comment:
Added logging to `Module`

Makes it much easier to tell how long each builder step is taking.